lib/datasource/aws-machine-image/index.spec.ts

```diff
@@ -0,0 +1,296 @@
+import {
+  DescribeImagesCommand,
+  DescribeImagesResult,
+  EC2Client,
+  Image,
+} from '@aws-sdk/client-ec2';
+import { mockClient } from 'aws-sdk-client-mock';
```

Comment:
@HonkingGoose  Can you open a new issue to refactor aws tests, so all tests which use the aws-sdk are using the `mockClient` or not

https://github.com/renovatebot/renovate/blob/b4a84065a1e658a7be637c7b381d8828dea80a3a/lib/datasource/docker/index.spec.ts#L18

Reply:
Made a new issue, see: #12739